scheduler/kubernetes/snaps.md

```diff
@@ -17,27 +18,27 @@ This document will show you how to create snapshots of Portworx volumes and how
 
 ## Snapshot types
 
-Using STORK, you can take 3 types of snapshots: 
-1. [In-cluster](/scheduler/kubernetes/snaps-in-cluster.html): These are per volume snapshots where the snapshots are stored in the current Portworx cluster's storage pools.
+Using STORK, you can take 2 types of snapshots:
+
+1. [Local](/scheduler/kubernetes/snaps-local.html): These are per volume snapshots where the snapshots are stored locally in the current Portworx cluster's storage pools.
 2. [Cloud](/scheduler/kubernetes/snaps-cloud.html): These snapshots are uploaded to the configured S3-compliant endpoint (e.g AWS S3).
-3. [Group](/scheduler/kubernetes/snaps-group.html): These allow you to snap multiple volumes at the same consistency point.
 
-3DSnaps is the umbrella term that covers PX-Enterprise's capability to provide app-consistent cluster wide snapshots on any node or cloud. 
+3DSnaps is the umbrella term that covers PX-Enterprise's capability to provide app-consistent cluster wide snapshots whether they are local or cloud.
```

Comment:
This will only be supported for local snaps right now. Maybe we should make that clear here.

Reply:
done